casper/src/test/scala/coop/rchain/casper/api/CreateBlockAPITest.scala

```diff
@@ -1,265 +0,0 @@
-package coop.rchain.casper.api
```

Comment:
Are there replacement tests for create block API?

Reply:
https://github.com/rchain/rchain/blob/34146e9a8f29cd49fe2a814ba7d0a78fe5f69bd6/casper/src/test/scala/coop/rchain/casper/addblock/ProposerSpec.scala#L25 is the replacement

Reply:
I don't see that BlockAPI is used at all in Proposer tests. I'm I missing something?

Reply:
I believe how CreateBlockAPITest is written - it is a test for proposer logic not for API. We might write another test for blockAPI to test if it is returning valid responses. Maybe do this after this <https://github.com/rchain/rchain/blob/956680dbef1063e518912405f909618be28d0570/casper/src/main/scala/coop/rchain/casper/api/BlockAPI.scala#L118-L120>?

Reply:
No doubt we need to redefine block API. I'm ok to leave this as part of those changes.